lua/entities/seamless_portal.lua

```diff
@@ -124,20 +124,26 @@ local function DrawQuadEasier(e, multiplier, offset, rotate)
 
 	local pos = e:GetPos() + ox + oy + oz
 	if rotate == 0 then
+		-- What the fuck is this?
+		-- TODO: Comment what this does
```

Comment:
all of these are drawquad funcs, basically it just draws a quad relative to the portal depending on a defined direction (var rotate)